icepyx/core/query.py

```diff
@@ -531,6 +140,54 @@ def tracks(self):
         else:
             return sorted(set(self._tracks))
 
+    def _get_concept_id(self, product, version) -> Union[str, None]:
+        collections = earthaccess.search_datasets(
+            short_name=product, version=version, cloud_hosted=True
+        )
+        if collections:
+            return collections[0].concept_id()
+        else:
+            return None
+
+    @property
+    @deprecated(
+        version="1.4.0", reason="order_vars() is going away, use variables() instead"
+    )
+    def order_vars(self) -> Union[Variables, None]:
+        """This used to print the list of vasriables for subsetting, Harmony doesn't provide that for IS2 datasets.
+        we do need to implement a class that gets the variables even if it'sm only for listing.
+        """
+        logging.warning(
+            "Deprecated: order_vars() is going away, use variables() instead"
+        )
+        if self.product:
+            self._variables = Variables(product=self.product)  # type: ignore[no-any-return]
+            return self._variables
+        return None
+
+    @property
+    def variables(self) -> Variables:
+        if not hasattr(self, "_variables"):
+            self._variables = Variables(product=self.product)
+        return self._variables
+
+    def show_custom_options(self) -> None:
+        """
+        Display customization/subsetting options available for this product.
+
+        Parameters
+        ----------
+        dictview : boolean, default False
+            Show the variable portion of the custom options list as a dictionary with key:value
+            pairs representing variable:paths-to-variable rather than as a long list of full
+            variable paths.
+
```

Comment:
```suggestion
```

Reply:
We might be able to grab some of the examples from the docstring of the original function too.